src/components/SingleArticle.jsx

```diff
@@ -1,30 +1,109 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { getArticle } from "../../api";
+import * as React from 'react';
+import { getArticle, getComments, getUsers } from "../../api";
 import Box from '@mui/material/Box';
 import {drawerWidth} from '../components/Header'
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import Avatar from '@mui/material/Avatar';
+import Typography from '@mui/material/Typography';
+import Divider from '@mui/material/Divider';
 
-export default function SingleArticle({articles}) {
+
+export default function SingleArticle({users, setUsers}) {
```

Comment:
Ideally we want a loading state here to update us if the api data is still loading